sunpy/net/dataretriever/sources/kanzelhohe.py

```diff
@@ -0,0 +1,128 @@
+        """
+        Return list of URLS corresponding to value of input timerange.
+
+        Parameters
+        ----------
+        timerange: `~sunpy.time.TimeRange`
+            time range for which data is to be downloaded.
+
+        Returns
+        -------
+        urls : `list`
+            list of URLs corresponding to the requested time range
+        """
+        wave = kwargs['wavelength']
+        wave_float = (wave.min if isinstance(wave, a.Wavelength) else wave.min()).value
+        table = {6563: {'datatype': ['halpha2k/recent', 'halph_fr'],
+                        'start_time': datetime.datetime(2000, 7, 20, 7, 45, 46)},
+                 32768: {'datatype': ['caiia', 'caiik_fi'],
+                         'start_time': datetime.datetime(2000, 7, 20, 7, 45, 46)},
+                 5460: {'datatype': ['phokada', 'bband_fi'],
+                        'start_time': datetime.datetime(2000, 7, 20, 7, 45, 46)}}
+        # Checking if value is close enough to a wavelength value.
+        # Converting from one unit to other introduces precision errors.
+        wavelengths = table.keys()
+        wave_list = list(filter(lambda x: np.isclose(x, wave_float), wavelengths))
+        if(len(wave_list) == 0):
+            raise ValueError("no value is enough closer to allowed wavelengths: "
+                             + ",".join(map(str, table.keys())) + " angstroms")
+        wave = wave_list[0]
+        start_date = table[wave]['start_time']
+        if timerange.start < start_date:
+            msg = 'Earliest date for which Kanzelhohe data is available is {:%Y-%m-%d}'
+            raise ValueError(msg.format(start_date))
+        prefix = "http://cesar.kso.ac.at/{datatype}/%Y/"
+        suffix = ""
+        if wave != 6563:
+            suffix = "%Y%m%d/processed/"
+        url_pattern = prefix + suffix + "kanz_{datatype1}_%Y%m%d_%H%M%S.fts.gz"
+        dtypes = table[wave]['datatype']
+        scraper = Scraper(url_pattern, datatype=dtypes[0], datatype1=dtypes[1])
+        if not timerange:
+            return []
+        result = scraper.filelist(timerange)
+        return result
+
+    def _makeimap(self):
+        """
+        Helper Function:used to hold information about source.
+        """
+        self.map_['source'] = 'KSO'  # TODO: check with kanzelhohe
+        self.map_['instrument'] = 'Kanzelhohe HA2'
+        self.map_['phyobs'] = 'irradiance'
+        self.map_['provider'] = 'KSO'
+
+    @classmethod
+    def _can_handle_query(cls, *query):
+        """
+        Answers whether client can service the query.
+
+        Parameters
+        ----------
+        query : list of query objects
+
+        Returns
+        -------
+        boolean: answer as to whether client can service the query
+        """
+        chk_var = 0
+        for x in query:
+            if x.__class__.__name__ == 'Instrument' and isinstance(
+                    x.value, str) and x.value.lower() == 'kanzelhohe':
```

Comment:
To keep everyone happy and since [wikipedia](https://en.wikipedia.org/wiki/Kanzelhoehe_Solar_Observatory), their official website (or so [it says google](https://www.google.com/search?q=kanzelhoehe+site%3Awww.kso.ac.at&oq=kanzelhoehe+site%3Awww.kso.ac.at), I've not found where exactly) and [their (?) Facebook group](https://www.facebook.com/pages/Kanzelhoehe-Solar-Observatory/127081834002472) puts the ö as oe I would suggest we accept the three of them as:

```suggestion
                    x.value, str) and x.value.lower() in ['kanzelhohe', 'kanzelhoehe', 'kanzelhöhe']:
```

I don't know German, but it seems it's a common practice to substitute the ö by oe when they don't have a keyboard layout that allows to put the umlaut.